src/auth/auth.module.ts

```diff
@@ -0,0 +1,20 @@
+import { Module } from '@nestjs/common';
+import { AuthService } from './auth.service';
+import { AuthController } from './auth.controller';
+import { UserModule } from './user/user.module';
+import { JwtModule } from '@nestjs/jwt';
+
+@Module({
+  imports: [
+    UserModule,
+    JwtModule.register({
+      global: true,
+      secret: `${process.env.JWT_SECRET}`,
+      signOptions: { expiresIn: '2h' },
```

Comment:
Add Zod schema validation for ENV values here.
Add JWT_EXPIRATION in env.

Reply:
done with joi